common/src/main/scala/org/apache/celeborn/common/meta/WorkerInfo.scala

```diff
@@ -186,7 +186,9 @@ class WorkerInfo(
       val curDisk = diskInfos.get(mountPoint)
       if (curDisk != null) {
         curDisk.actualUsableSpace_$eq(newDisk.actualUsableSpace)
-        curDisk.activeSlots_$eq(Math.max(curDisk.activeSlots, newDisk.activeSlots))
+        // Do not update active slots by worker heartbeat message
+        // because partitions identified as unknown worker will be filled
+        // into disk or HDFS but master won't know it.
```

Comment:
The sentence needs to be polished, TBH, I can not understand it, even though I know the background